perception/compare_map_segmentation/src/voxel_based_approximate_compare_map_filter_nodelet.cpp

```diff
@@ -28,6 +28,16 @@ VoxelBasedApproximateCompareMapFilterComponent::VoxelBasedApproximateCompareMapF
   const rclcpp::NodeOptions & options)
 : Filter("VoxelBasedApproximateCompareMapFilter", options)
 {
+  // initialize debug tool
+  {
+    using tier4_autoware_utils::DebugPublisher;
+    using tier4_autoware_utils::StopWatch;
+    stop_watch_ptr_ = std::make_unique<StopWatch<std::chrono::milliseconds>>();
+    debug_publisher_ =
+      std::make_unique<DebugPublisher>(this, "voxel_based_approximate_compare_map_filter");
+    stop_watch_ptr_->tic();
```

Comment:
is it better?
```suggestion
    stop_watch_ptr_ = std::make_unique<tier4_autoware_utils::StopWatch<std::chrono::milliseconds>>();
    debug_publisher_ =
      std::make_unique<tier4_autoware_utils::DebugPublisher>(this, "voxel_based_approximate_compare_map_filter");
    stop_watch_ptr_->tic();
```

Reply:
@yukkysaito 
I'd like to use using because 
- this part is isolated with `{ }` so affect of using is limited to this part.
- formatter changes  indentation and that causes less readability

Reply:
Ok 👍 
It may be a matter of taste.